python/ray/rllib/models/modelv2.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from collections import namedtuple
+from ray.rllib.models.model import restore_original_dimensions
+from ray.rllib.utils import try_import_tf
+
+tf = try_import_tf()
+
+
+class OutputSpec(namedtuple("OutputSpec", ["size"])):
```

Comment:
I think I forgot to mention this during our last discussion but this feels a bit unnecessary. Is there any reason why an `int` couldn't replace it everywhere it is used?

Something like this could be useful to have throughout rllib, but it might be best to make it more informative with more utility (e.g., `OutputSpec.is_shape_compatible(...)`) to warrant the added complexity (I'm thinking something analogous to `tf.TensorSpec`). Otherwise, I feel like a basic `int` (and/or a scalar `tf.Tensor`) would work just as well while being more streamlined.

Reply:
Agreed, I'll remove it along with the other changes discussed.